<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1441</width>
      <height>781</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="acceuil">
     <attribute name="title">
      <string>acceuil</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="categorie">
     <attribute name="title">
      <string>categorie</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="clients">
     <attribute name="title">
      <string>client</string>
     </attribute>
     <widget class="QCommandLinkButton" name="commandLinkButton_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>520</y>
        <width>187</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer client</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>400</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ajouter client</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>161</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255,1);
border:none;
heigth:60%;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="styleSheet">
          <string notr="true">.QPushButton {

  border-radius: 25%;
  background-color: #f4511e;

  color: #FFFFFF;
  text-align: center;
  font-size: 15px;
  padding: 20px;


  cursor: pointer;
  margin: 5px;
}

.QPushButton:hover  {
 padding-right: 25px;
font-size: 22px;
background-color:black;
}

</string>
         </property>
         <property name="text">
          <string>client</string>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="styleSheet">
          <string notr="true">.QPushButton {

  border-radius: 25%;
  background-color: #f4511e;

  color: #FFFFFF;
  text-align: center;
  font-size: 15px;
  padding: 20px;


  cursor: pointer;
  margin: 5px;
}

.QPushButton:hover  {
  padding-right: 25px;
font-size: 22px;
 background-color:black;
}

</string>
         </property>
         <property name="text">
          <string>fidélité</string>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tabclient">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>621</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>120</y>
        <width>321</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>cin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>prenom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>tel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>age</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditcin">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditnom">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditprenom"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdittel"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditage"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_3">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>400</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>modifier client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditrecherche">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>252</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditcins">
      <property name="geometry">
       <rect>
        <x>711</x>
        <y>530</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>520</y>
        <width>56</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>37</y>
        <width>71</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>73</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>age</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tel</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>37</y>
        <width>71</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>trier par</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="employes">
     <attribute name="title">
      <string>employe</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="assurance">
     <attribute name="title">
      <string>assurance</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="location">
     <attribute name="title">
      <string>location</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="marketing">
     <attribute name="title">
      <string>marketing</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
